.github/workflows/ci.yml

```diff
@@ -173,26 +173,12 @@ jobs:
         run: npm run build:dev
       - name: Test types
         run: npm run test:types
-      # On Linux we run all Chrome tests without retries and Firefox tests with retries.
-      - name: Run all Chrome tests (only on Linux)
+      - name: Run all tests with xvfb
```

Comment:
+1 to run in headful mode as well

Reply:
Could you clarify? we have been already running the headful mode tests for a while (no change in behaviour here)

Reply:
I wasn't aware of that, and it would be useful in Mapper as well.